access-control-service-sql-impl/pom.xml

```diff
@@ -92,6 +100,23 @@
          <groupId>org.slf4j</groupId>
          <artifactId>slf4j-simple</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.mockito</groupId>
+         <artifactId>mockito-core</artifactId>
+         <scope>test</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.hibernate.validator</groupId>
+         <artifactId>hibernate-validator</artifactId>
+         <version>8.0.1.Final</version>
```

Comment:
Please move the version to the parent pom.

Reply:
Done, thank you!